### PR TITLE
cmf 1.3 release note

### PR DESCRIPTION
**Pull request by dbu**
in large parts, this news is a copy of the RC announcement. but i don't think there is much else to say, and its more readable this way than linking to the other news for feature list or the future.

but feel free to correct/add/remove things. would be great to get this merged fast so that we can use it to announce the release.

### CODE REVIEW

**Comment by ElectricMaxxx** [comment]
@dbu maybe adding some Words to the MenuBundle blockig KPNMenuBundle Rafts, that we solved.

File: src/AppBundle/Resources/data/posts/2016-04-04.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    The version numbers for each bundle are tracked separately, which explains
+    why not all of them are synchronized.
+
+    ### Important new Features
+
+    The most important features included in these new versions are:
+
+     * PHP 7 support for all CMF packages;
+     * The Routing and Testing components support Symfony 3;
+     * The CmfMenuBundle added KnpMenu 2 support and dropped support for KnpMenu 1;
```
Comment (dbu) [inline src/AppBundle/Resources/data/posts/2016-04-04.yml]: @ElectricMaxxx here we mention knp menu 2. is that what you meant?

Comment (ElectricMaxxx) [inline src/AppBundle/Resources/data/posts/2016-04-04.yml]: Oh. Ups. Not seen. Sorry. :-)

Comment (ElectricMaxxx) [inline src/AppBundle/Resources/data/posts/2016-04-04.yml]: But then it looks nice from my POV.

**Comment by dbu** [comment]
wouter, can you merge when you are happy too?

i should be able to tweet with the cmf twitter account tomorrow once this is online.

**Pull request merged**